scripts/mkcloud

```diff
@@ -199,6 +201,21 @@ function error_exit()
     exit $exitcode
 } >&2
 
+function determine_host_mtu()
+{
+    local host_mtu
+
+    host_mtu=$(LC_ALL=C sort -n /sys/class/net/*/mtu | head -n 1)
+    # We may also have want_mtu_host so in that case we need to adjust
+    # it accordingly
+    if [[ $want_mtu_size -gt $host_mtu ]]; then
+        echo "Lowering vNIC MTU from $want_mtu_size to $host_mtu..."
+        want_mtu_size=$host_mtu
```

Comment:
But can you use jumbo frames if the host NIC only has MTU 1500 or lower? well you could, but that you essentially render all traffic forwarded by the host NIC useless. How do you deal with that at the moment? Because I believe that your hypervisor sits on an regular Ethernet network with 1500 MTU.